app/components/Rating/Rating.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { RatingTypes } from "./Rating.types";
+import classNames from "classnames";
+
+const Rating = ({ value, variant, extraClasses }: RatingTypes) => {
+  const ratingClasses = classNames({
+    "w-10 bg-lightBlack rounded-full h-10 text-white text-center p-2 text-sm font-bold absolute top-[200px] left-8 border-2":
```

Comment:
the absolute styles should not be added in the component like this, instead you should pass these as extraClasses.
Reason: This component might be used somewhere else where this posirion is not required